loki/visitors/ir_graph.py

```diff
@@ -184,9 +184,33 @@ def __add_node(self, node, **kwargs):
             A list of a tuple of a node and potentially a edge information
         """
         label = kwargs.get("label", "")
+
         if label == "":
             label = self.format_node(repr(node))
 
+        try:
+            live_symbols = "live: [" + ", ".join(
+                str(symbol) for symbol in node.live_symbols
+            )
+            defines_symbols = "defines: [" + ", ".join(
+                str(symbol) for symbol in node.defines_symbols
+            )
+            uses_symbols = "uses: [" + ", ".join(
+                str(symbol) for symbol in node.uses_symbols
+            )
+            label = self.format_line(
+                label,
+                "\n",
+                live_symbols,
+                "], ",
+                defines_symbols,
+                "], ",
+                uses_symbols,
+                "]",
+            )
+        except (RuntimeError, KeyError, AttributeError) as _:
```

Comment:
I think you can simply skip the `as _` if you're not planning to use the exception